orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/RestrictExecutionDuringTimeWindow.groovy

```diff
@@ -41,8 +44,28 @@ class RestrictExecutionDuringTimeWindow implements StageDefinitionBuilder {
   public static final String TYPE = "restrictExecutionDuringTimeWindow"
 
   @Override
+  @CompileStatic(TypeCheckingMode.SKIP)
   void taskGraph(Stage stage, TaskNode.Builder builder) {
     builder.withTask("suspendExecutionDuringTimeWindow", SuspendExecutionDuringTimeWindowTask)
+    if (stage.context.restrictedExecutionWindow?.splay?.enabled && stage.context.restrictedExecutionWindow?.splay?.maxDelay) {
+      if (stage.context.restrictedExecutionWindow.splay.skipManual && stage.execution.trigger?.type == "manual") {
+        return
+      }
+
+      int min = stage.context.restrictedExecutionWindow.splay.minDelay ?: 0
+      int max = stage.context.restrictedExecutionWindow.splay.maxDelay
+      int waitTime = ThreadLocalRandom.current().nextInt(min, max + 1)
+
```

Comment:
Inject a WaitTask to randomly pad the start of a RestrictedExecutionWindow within an optional lower and required upper bound, measured in seconds. If `skipManual==true` (this will likely be checked by default in deck) the WaitTask is skipped if the pipeline was manually triggered.

This extends stage `restrictedExecutionWindow` as below, deck PR to follow.
```
      "restrictedExecutionWindow": {
        "splay": {
          "enabled": true,
          "maxDelay": 300,
          "minDelay": 30,
          "skipManual": false
        },
        "whitelist": [
          {
            "endHour": 18,
            "endMin": 0,
            "startHour": 14,
            "startMin": 30
          }
        ]
      },
```